runtime/development/src/lib.rs

```diff
@@ -2127,6 +2142,7 @@ construct_runtime!(
 		ForeignInvestments: pallet_foreign_investments::{Pallet, Storage, Event<T>} = 117,
 		OraclePriceFeed: pallet_oracle_feed::{Pallet, Call, Storage, Event<T>} = 118,
 		OraclePriceCollection: pallet_oracle_collection::{Pallet, Call, Storage, Event<T>} = 119,
+		AnchorsV2: pallet_anchors_v2::{Pallet, Call, Storage, Event<T>} = 130,
```

Comment:
Are we ok with this index?

Reply:
I'll place it where we write: `// out pallets part 2` to no mix indexes